.github/workflows/deployment_plan.yaml

```diff
@@ -19,15 +19,15 @@ on:
         description: 'dibbs-ecr-viewer docker image tag'
         required: true
         type: string
-        default: "v2.0.0-beta"
+        default: "3.4.0"
```

Comment:
I have a question. This likely isn't blocking, but it's something to potentially consider for future refinements. With the new variables and config names in here, is there now a "version floor" for the eCR Viewer that this module will not work with if we go below? I'm wondering if we need to add in a check somewhere to keep folks from downgrading past a certain point.

Reply:
Great question and suggestion! We likely have a version floor because the Environment variables for the app have changed several times. I'll reach out to the devs, and we can compile a list of versions that should work.